mackerel.go

```diff
@@ -66,6 +67,12 @@ func NewClientWithOptions(apikey string, rawurl string, verbose bool) (*Client,
 	}, nil
 }
 
+// SetMaxRetries configures this client to retry HTTP request up to *max* times
+func (c *Client) SetMaxRetries(max int) *Client {
```

Comment:
I think we should use a method name by which we can understand this is retry count for network error.  For example, `SetRetryCountOnNetworkError`